class.json-api.php

```diff
@@ -616,6 +616,84 @@ function get_avatar_url( $email, $avatar_size = null ) {
 		}
 	}
 
+	/**
+	 * Counts the number of comments on a site, excluding certain comment types.
+	 *
+	 * @param $post_id int Post ID.
+	 * @return array Array of counts, matching the output of https://developer.wordpress.org/reference/functions/get_comment_count/.
+	 */
+	public function wp_count_comments( $post_id ) {
+		global $wpdb;
+		if ( 0 !== $post_id ) {
+			return wp_count_comments( $post_id );
+		}
+
+		$counts = array(
+			'total_comments' => 0,
+			'all'            => 0,
+		);
+
+		/**
+		 * Exclude certain comment types from comment counts.
+		 *
+		 * @since 6.8.1
```

Comment:
Could you change this for `6.9.0` instead? I would also recommend adding `@module json-api` to the docblock, it helps our parser assign each hook to the right category in our Codex.